src/Command/Route/ListCommand.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Command\Route;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Yiisoft\Router\Route;
+use Yiisoft\Router\UrlMatcherInterface;
+use Yiisoft\Yii\Console\ExitCode;
+
+class ListCommand extends Command
+{
+    private UrlMatcherInterface $urlMatcher;
+
+    protected static $defaultName = 'route/list';
```

Comment:
I think names `router/list` or `routes/list` woulbe be more relevant

Reply:
Probably the name `routes / list` might be appropriate, but we accept the singular instead of the plural.

Reply:
How about `router/list`? In the future it can be multiple commands like the following: `router/match`, `router/group`, `router/middlewares`, `router/handler` and etc.

Reply:
Done.